Userland/Libraries/LibJS/Runtime/TypedArray.h

```diff
@@ -452,6 +452,7 @@ class TypedArray : public TypedArrayBase {
 
 ThrowCompletionOr<TypedArrayBase*> typed_array_create(GlobalObject& global_object, FunctionObject& constructor, MarkedVector<Value> arguments);
 ThrowCompletionOr<double> compare_typed_array_elements(GlobalObject& global_object, Value x, Value y, FunctionObject* comparefn, ArrayBuffer&);
+ThrowCompletionOr<TypedArrayBase*> typed_array_create_same_type(GlobalObject& global_object, TypedArrayBase const& exemplar, MarkedVector<Value> arguments);
```

Comment:
let's also keep the declarations in order - `typed_array_create`, then `typed_array_create_same_type` :)